app/admin/celebrate_shoutouts.rb

```diff
@@ -11,7 +11,7 @@
     actions
   end
 
-  filter :user, as: :select, collection: User.all.map { |item| [item.full_name, item.id] }, label: 'Sender'
-  filter :time_period, as: :select, collection: TimePeriod.all.map { |item| [item.date_range, item.id] }
-  filter :recipients,  as: :select, collection: User.all.map { |item| [item.full_name, item.id] }
+  filter :user, as: :select, collection: User.all.order(:email).map { |u| ["#{u.first_name} #{u.last_name}", u.id] }, label: 'Sender'
+  filter :time_period, as: :select, collection: TimePeriod.all.order(start_date: :desc).map { |t| [t.date_range, t.id] }
+  filter :recipients,  as: :select, collection: User.all.order(:email).map { |r| [r.full_name, r.id] }
```

Comment:
@LyubovP in the User model we have method `full_name` for displaying first name and last name, please try use it. Also you can remove `.all` after the User, because you use method `order`. Please check the same places in PR.

Reply:
Thank you for the information. I will check.